docs/yaml/elementary/str.yml

```
@@ -41,6 +41,35 @@ methods:
     description: The values to replace the @number@ placeholders in the format string.
     type: int | bool | str
 
+# str.hash(hash_algo)
+- name: hash
+  description: Run the given `hash_algo` on the string and return the hash constant
+  returns: str
+  example: |
+    ```meson
+        message('foobar'.hash('md5'))    # 3858f62230ac3c915f300c664312c63f
```

```suggestion
        message('foobar'.hash('md5'))    # => '3858f62230ac3c915f300c664312c63f'
```
to be consistent with other annotations in this file.